desktop/settings/linux-settings.md

```diff
@@ -0,0 +1,151 @@
+---
+description: Docker Desktop settings
+keywords: Settings, Docker Desktop, Linux
+title: Change Docker Dektop settings on Linux
+---
+
+This page provides information on how to configure and manage your Docker Desktop settings.
+
+To navigate to **Settings** either:
+
+- Select the Docker menu ![whale menu](../images/whale-x.png){: .inline} and then **Settings** 
+- Select the **Settings** icon from the Docker Dashboard. 
+
+## General
+
+On the **General** tab, you can configure when to start Docker and specify other settings:
+
+- **Start Docker Desktop when you log in**. This automatically starts Docker Desktop when you open your session.
+
+- **Send usage statistics**. Docker Desktop sends diagnostics, crash reports, and usage data. This information helps Docker improve and troubleshoot the application. Clear the check box to opt out.
```

Comment:
Looks like a lot of this content is duplicated between Linux, Mac, and Windows (but sometimes phrased _slightly_ different).

Ideally, we would either combine the pages (and call out some options to be Mac-, Windows- or Linux-specific, or move common content to a common page.

To help finding the differences, perhaps we should do a quick sync between the pages to make the common content align (that way a "compare" between the pages shows exactly what sections are different).